### PR TITLE
copytranslator: Add version 9.0.0

### PR DESCRIPTION
**Pull request by Dothion**
First time pr for scoop bucket, please check twice😂

### CODE REVIEW

**Comment by Dothion** [comment]
#81

File: bucket/copytranslator.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "homepage": "https://copytranslator.github.io/",
+    "description": "Foreign language assisted reading and translation solution",
+    "license": "GPLv2",
```
Comment (h404bi) [inline bucket/copytranslator.json]:
```suggestion
    "license": "GPL-2.0-only",
```

File: bucket/copytranslator.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "url": "https://github.com/CopyTranslator/CopyTranslator/releases/download/v9.0.1/copytranslator-9.0.1-win.zip",
+    "hash": "e56c98df1c86e74feac07b9b0df0a0282dcb174ffeefd82e4a75fba488742a43",
+    "bin": "copytranslator.exe",
+    "shortcuts": [
+        [
+            "copytranslator.exe",
+            "CopyTranslator"
+        ]
+    ],
+    "checkver": "github",
```
Comment (h404bi) [inline bucket/copytranslator.json]:
```suggestion
    "checkver": {
        "github": "https://github.com/CopyTranslator/CopyTranslator"
    },
```

File: bucket/copytranslator.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+    "homepage": "https://copytranslator.github.io/",
+    "description": "Foreign language assisted reading and translation solution",
+    "license": "GPLv2",
+    "version": "9.0.1",
+    "url": "https://github.com/CopyTranslator/CopyTranslator/releases/download/v9.0.1/copytranslator-9.0.1-win.zip",
+    "hash": "e56c98df1c86e74feac07b9b0df0a0282dcb174ffeefd82e4a75fba488742a43",
+    "bin": "copytranslator.exe",
```
Comment (h404bi) [inline bucket/copytranslator.json]:
```suggestion
```

**Comment by h404bi** [comment]
I've noticed v9.0.1 is a pre-release version, once the pr get merged the automation will roll it back to v9.0.0 since v9.0.0 the latest release instead.

**Comment by h404bi** [comment]
Please update the URL and hash, too.

**Comment by h404bi** [comment]
Thanks for your contribution.

**Pull request merged**